scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -4442,3 +4489,31 @@ function loaded() {
         EventBridge.emitWebEvent(JSON.stringify({ type: 'propertiesPageReady' }));
     }, 1000);
 }
+
+function showOnTheSamePage(entityType) {
+    var i;
+    var numbType = entityType.length;
+    var matchingType = 0;
+    for (i = 0; i < numbType; i++) {
+        if (GROUPS_PER_TYPE[entityType[i]].includes(currentTab)) {
```

Comment:
I am confused about what you're expecting out of `entityType[i]`  ,    since the input `entityType` seems to be a string

Reply:
"entityType" is an array. The list of the different types found among the selected entities.
The function detect if the current tab is part of each group of the listed type.
If it is not matching, this means that the current tab is not common to all the selected entities. In that case, we force the "base" tab which is always common. (Initially, I thought it was a string too. This was the cause of the bug for the multiple selections.